### ISSUE TITLE
JUnit Test Result Trend visualization Problem 

### ISSUE THREAD

**Issue by ShiqiHe000** [nextest-rs/nextest#1158]
Hey! I'm using `cargo-nextest 0.9.64` to output the test results in Junit XML files and display it in my Jenkins CI pipeline using JUnit plug-in.

I found 2 problems with the Junit output:
- No show of "skipped tests" in Junit.xml when setting `status-level = "all"` in `nextest.xml`, which is an open issue [here](https://github.com/nextest-rs/nextest/issues/885). 
- The scaling in the Test result trend graph is wrong. 

To be specific of the second issue: I have a Jenkins pipeline which use `nextest` to generate the unit XML file (`cargo nextest run --workspace --no-fail-gast --profile ci`). In the `nextest.toml` file I have 
```
...
status-level = "all"

[profile.ci.junit]  # this can be some other profile, too
path = "junit.xml" # this will generate the xml file under coverage_reports/nextest/ci/junit.xml
store-success-output = true
store-failure-output = true
...
```

After several builds, I can have a Test Result Tend Graph in the Jenkins UI: 
![image](https://github.com/nextest-rs/nextest/assets/53793150/414ec88f-d539-4303-a6f7-173de70459dd)

For build #7, 8, 9, passed tests:2817, failed: 17, skipped: 0
![image](https://github.com/nextest-rs/nextest/assets/53793150/963c72df-3c73-45ed-a8f5-c61f1ba7aee4)

For build #10, passed tests: 2803. failed: 31, skipped: 0
![image](https://github.com/nextest-rs/nextest/assets/53793150/4cd74fa8-7cd6-4c64-8ceb-66b77fd42fd4)

As you can see, the passed/failed tests display area ratio is off. From the graph, it looks like there are more failed tests (red area) than passed tests (blue area) 😨. I know the number starts with 2803 on the y-axis and there is some scaling happening at the bottom: the number increments by 3 then by 5. But this graph makes no sense to me as it doesn't show the real relative ratio between passed and failed tests. ☹️

If I only show one factor in the graph it makes more sense, for example, only shows "failed tests":
![image](https://github.com/nextest-rs/nextest/assets/53793150/343ba3c8-779e-45ae-9af5-1e49b24b50c4)

Or "passed tests":
![image](https://github.com/nextest-rs/nextest/assets/53793150/7add3aa6-4427-43d1-a488-c7cdd0eff861)

Is there anything we can do in `nextest` to make the trend graph to do a better job? Thank you! 😀

**Comment by sunshowers** [comment]
Thanks for the report. That looks like Jenkins is producing a graph that's correct, but has a non-zero Y axis. This is sometimes useful but often misleading (as you've discovered here).

I don't think there's anything nextest can do since we're just producing output in the right format for it. This seems like a Jenkins presentation issue.

**Comment by ShiqiHe000** [comment]
Okay got it. Thanks for the help 😄

**Comment by sunshowers** [comment]
The other issue is covered by #885, so I'm going to close this. (If you'd like to contribute a fix to #885 it would be greatly appreciated!)
